common/constants/stats.js

```diff
@@ -181,6 +181,24 @@ export const statsData = {
     suffix: "",
     color: "5",
   },
+  fishing_speed: {
+    name: "Fishing Speed",
+    nameLore: "Fishing Speed",
+    nameShort: "Fishing Speed",
+    nameTiny: "FS",
+    symbol: "☂",
+    suffix: "",
+    color: "b"
+  },
+  wisdom: {
```

Comment:
interesting. I would have thought wisdom was already there

Reply:
Same, not sure why it wasn't 🤷

Reply:
Wisdom is new, it was added to replace XXX skill xp boost